src/shell/store/ui.ts

```diff
@@ -155,6 +155,14 @@ const ICON_CONFIG: { [index: string]: SvgIconComponent } = Object.freeze({
 
 // Thunk helper functions
 export function tabLocationEquality(tab1: TabLocation, tab2: TabLocation) {
+  // Makes sure that a new tab isn't created when the user searches for something and only the filters are changed in the params
```

Comment:
So, for example we are on `/search?q=iamaquery` and that get's pinned, if I go ahead and add some filters which will then add more query params making it for example `/search?q=iamaquery&datePreset=last_7_days`, this will create a new tab instead of just sticking to the already open tab which was already querying the "iamaquery" keyword.

Reply:
Basically, this just makes sure that as long as we're already on the `/search` page and the query param `q` is the same, regardless if other query params are added, prevent creating another tab.

Reply:
I think I remember Zosh and I discussing this a while back that's why I went ahead and added it. I can confirm with him though.